cmd/manager/main.go

```diff
@@ -87,6 +96,21 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Setup event recorder
+	kubeCfg, err := clientcmd.BuildConfigFromFlags("", kubeConfigPath)
+	if err != nil {
+		setupLog.Error(err, "error creating rest config")
+		os.Exit(1)
+	}
+	kubeClient, err := kubernetes.NewForConfig(kubeCfg)
```

Comment:
we can use the following code:
```golang
	config := ctrl.GetConfigOrDie()
	kubeClient, err := kubernetes.NewForConfig(config)
	if err != nil {
		setupLog.Error(err, "error creating internal group client")
		os.Exit(1)
	}
	eventBroadcaster := record.NewBroadcaster()
	eventBroadcaster.StartLogging(glog.Infof)
	eventBroadcaster.StartRecordingToSink(&clientv1.EventSinkImpl{Interface: kubeClient.CoreV1().Events("")})


```

Reply:
also why do we need this eventBroadcaster here?

Reply:
We made a recorder from broadcaser